src/Client/WorkflowClient.php

```diff
@@ -54,7 +59,19 @@ public function __construct(
         $this->clientOptions = $options ?? new ClientOptions();
         $this->converter = $converter ?? DataConverter::createDefault();
         $this->starter = new WorkflowStarter($serviceClient, $this->converter, $this->clientOptions);
-        $this->reader = new WorkflowReader(new AttributeReader());
+        $this->reader = new WorkflowReader($this->createReader());
+    }
+
+    /**
+     * @return ReaderInterface
+     */
+    private function createReader(): ReaderInterface
```

Comment:
We need this method in the Annotations package. This is the default configuration for now.

Reply:
I agree. But for now, either so, or take it out separately as done in the Cycle ORM: https://github.com/cycle/annotated/blob/master/src/ReaderFactory.php